src/etc/inc/dyndns.class

```diff
@@ -2604,6 +2697,22 @@
 						$this->_debug($data);
 					}
 					break;
+				case 'hetzner':
+				case 'hetzner-v6':
+					$output = json_decode($data);
+					if ($output->record->value === $this->_dnsIP) {
+						$status = "Dynamic DNS: (Success) {$this->_dnsHost} updated to {$this->_dnsIP}";
+						$successful_update = true;
+					} elseif ($http_code == 401) {
+						$status = 'Dynamic DNS: (Error) Bad authentication attempt because of a wrong API Key.';
```

Comment:
Reword to fix grammar/phrasing:

`Bad authentication attempt because of a wrong API Key.` -> `Authentication failed, incorrect API Key`

Reply:
done